guacamole/src/main/webapp/app/manage/controllers/manageUserController.js

```diff
@@ -89,6 +97,18 @@ angular.module('manage').controller('manageUserController', ['$scope', '$injecto
     var username = $routeParams.id;
 
     /**
+     * The string value representing the user currently being edited within the
+     * permission flag set. Note that his may not match the user's actual
+     * username - it is a marker that is (1) guaranteed to be associated with
+     * the current user's permissions in the permission set and (2) guaranteed
+     * not to collide with any user that does not represent the current user
+     * within the permission set.
+     *
+     * @type String
+     */
+    var selfUsername = '';
```

Comment:
This is referenced as `$scope.selfUsername` elsewhere and used as `selfUsername` within the template. Probably shouldn't be `var`.